apps/web/src/utils/keyboard.ts

```diff
@@ -19,16 +19,24 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import EventManager from "@notesnook/core/utils/event-manager";
 
-const GlobalKeyboard = {};
+const GlobalKeyboard = {
+  addEventListener,
+  removeEventListener
+};
```

Comment:
This makes no sense. Why not define the functions in the object directly?

Like so:
```ts
const GlobalKeyboard = {
  addEventListener: () => ...,
  removeEventListener: () => ...
};
```

Reply:
Yes we can do it....

Reply:
It will be more clear and better